### PR TITLE
RS: OS rolling upgrade

### PR DESCRIPTION
**Pull request by rrelledge**
[DOC-1173](https://redislabs.atlassian.net/browse/DOC-1173) [DOC-901](https://redislabs.atlassian.net/browse/DOC-901)

[Staged preview](https://docs.redis.com/staging/DOC-2800/rs/installing-upgrading/upgrading/upgrade-os/)

[DOC-1173]: https://redislabs.atlassian.net/browse/DOC-1173?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-901]: https://redislabs.atlassian.net/browse/DOC-901?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cmilesb — APPROVED** [review]

**Comment by kurtfm** [comment]
@rrelledge this looks great.  I think we need to add to the 'Verify node health' what should they expect for success at a minimum when running rlcheck or rladmin extra all.

File: content/stack/install/add-module-to-cluster.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ To use other modules or upgrade an existing module to a more recent version, you
 
 To install or upgrade a module on a [Redis Enterprise]({{<relref "/rs">}}) cluster, you need a module package.
 
-- For Redis Enterprise modules, download packages from the [Redis download center](https://redislabs.com/download-center/modules/).
+- For the latest Redis Enterprise modules, download packages from the [Redis download center](https://redislabs.com/download-center/modules/).
+
+- For earlier versions of Redis Enterprise modules that are no longer available from the Redis download center, [contact support](https://redis.com/company/support/).
+
 - For custom-packaged modules, either download a [custom-packaged module](https://redislabs.com/community/redis-modules-hub/) from the developer or [package the module yourself]({{<relref "/stack/install/packaging-modules">}}).
```
Comment (kurtfm) [inline content/stack/install/add-module-to-cluster.md:31]: Do we support this or want people to do this?  I know for sure we don't support people creating and using their own modules.

Comment (kurtfm) [reply content/stack/install/add-module-to-cluster.md:31]: Maybe its necessary in certain cases when supporting a preview.  Not sure, I feel like we should confirm this.  Maybe I am off.

Comment (rrelledge) [reply content/stack/install/add-module-to-cluster.md:31]: @K-Jo @LiorKogan What are your thoughts on this? Should I remove the following statement from the Enterprise docs?

> For custom-packaged modules, either download a [custom-packaged module](https://redislabs.com/community/redis-modules-hub/) from the developer or [package the module yourself](https://docs.redis.com/latest/stack/install/packaging-modules/).

File: content/rs/installing-upgrading/upgrading/upgrade-os.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        ```sh
+        rlcheck
+        ```
+
+    1. Run [`rladmin status`]({{<relref "/rs/references/cli-utilities/rladmin/status">}}) on the new node:
+
+        ```sh
+        rladmin status extra all
+        ```
+
```
[the remark this answers is not in the document]
Comment (rrelledge) [reply content/rs/installing-upgrading/upgrading/upgrade-os.md:131]: Added

File: content/rs/installing-upgrading/upgrading/upgrade-os.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    rladmin cluster join nodes <cluster_member_ip_address> username <username> password <password> replace_node <node_id>
+    ```
+
+1. Verify node health:
+
+    1. Run `rlcheck` on all nodes:
+
+        ```sh
+        rlcheck
+        ```
```
[the remark this answers is not in the document]
Comment (rrelledge) [reply content/rs/installing-upgrading/upgrading/upgrade-os.md:108]: Added

**Pull request merged**